main.js

```diff
@@ -37,4 +37,60 @@ for (i = 0; i < arr3.length; i++) {
     if (arr3[i] % 2 == 0 && arr3[i] != 0) {
         console.log(arr3[i])
     }
-}
\ No newline at end of file
+}
+
+//lesson-3--1
+function palindrome(word) {
+    let arr = word.split('').reverse().join('');
```

Comment:
You don't plan to change `arr`, use  `const`.

Reply:
and result of `word.split('').reverse().join('');` will be `string`, not `arr`. wrong name for variable.

Reply:
done